server/server.go

```diff
@@ -189,26 +198,22 @@ func (cs *cribbageServer) ginGetPlayer(c *gin.Context) {
 }
 
 func (cs *cribbageServer) ginPostAction(c *gin.Context) {
-	action, err := unmarshalPlayerAction(c.Request)
+	reqBytes, err := ioutil.ReadAll(c.Request.Body)
 	if err != nil {
 		c.String(http.StatusBadRequest, `Error: %s`, err)
 		return
 	}
-
-	err = handlePlayerAction(action)
+	action, err := jsonutils.UnmarshalPlayerAction(reqBytes)
```

Comment:
Do we wanna bind here too? 🤷 either way

Reply:
Meh, I'm inclined to use the code we have if it's working :shrug: